frontend/src/components/Service/ServiceHeatmap.vue

```diff
@@ -82,18 +82,16 @@
       },
       methods: {
           async chartHeatmap() {
-              let start = new Date(new Date().getUTCFullYear(), new Date().getUTCMonth()-2, 1);
+              let start = new Date(Date.UTC(new Date().getUTCFullYear(), new Date().getUTCMonth()-2, 1));
```

Comment:
I think this can be simplified.

Reply:
Without the Date.UTC the date is not correctly initialized.
![image](https://user-images.githubusercontent.com/1446221/82125637-841c4800-97a7-11ea-9562-a7486ff64fb4.png)

![image](https://user-images.githubusercontent.com/1446221/82125662-af069c00-97a7-11ea-93ef-1265db6d937a.png)

Reply:
With simplified I mean less duplicated code parts and a few more variables ;-)